src/main/java/Archibald.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+
+public class Archibald {
+
+    private static Task[] tasks = new Task[100];
```

Comment:
Consider defining a constant for the array size. For example, ``private static final int MAX_TASKS = 100``